.github/workflows/ios-beta.yaml

```diff
@@ -0,0 +1,82 @@
+name: iOS Beta
+on:
+  push:
+    branches:
+      - master
+      - rt-ios-testflight
+    paths-ignore:
+      - .github/workflows/android*
+      - .github/workflows/linux*
+      - "**/*_tests/**"
+      - "**/CMakeLists.txt"
+      - CONTRIBUTORS
+      - LICENSE
+      - NOTICE
+      - README.md
+      - android/**
+      - docs/**
+      - generator/**
+      - packaging/**
+      - pyhelpers/**
+      - qt*/**
+      - skin_generator/**
+      - track_generator/**
+
+jobs:
+  apple-testflight:
+    name: Apple TestFlight
+    runs-on: macos-latest
```

Comment:
macos-11 не работает пока? Там все самые свежие версии лежат, нужные нам.

Reply:
Я попробую.

Reply:
Не для людей пока:

<blockquote>
A couple of months ago we added a new macOS 11.0 image in a “preview” mode and eventually we expect it to become a production image.

Typically, we increase pool capacity along with increasing image usage, but demand for this pool is outpacing our capacity plan. Given the constantly growing demand and in order to prevent degraded experience, we have to temporarily transit this pool to a private preview until we come up with a final plan of making macOS 11.0 ready for production use.

What does “private preview” mean? This limitation will live on organization level (personal accounts are considered as organization too), so if you are using macos-11 somewhere in your organization — you will still be able to use it. But this pool will be inaccessible for new organizations that never used it before.
</blockquote>

Reply:
Я попробую запросить в поддержке.